models/pipeline.js

```diff
@@ -112,7 +112,13 @@ const MODEL = {
     })
         .description('A list of badges that pipline has')
         .example(`{ sonar: { name: 'dashboard', uri: 'http://sonar.com/sample1' } }`)
-        .optional()
+        .optional(),
+
+    templateVersionId: Joi.number()
+        .integer()
+        .positive()
+        .description("Identifier for this pipeline's template")
+        .example(123345)
```

Comment:
do we need `optional` here? this is not mandatory field for pipeline object model, I believe.

Reply:
it was used for get validation thus we don't really have to set optional explicitly because it will always be there, but yeah we can add it anyway

Reply:
Schema would be optional by default